src/main/java/mate/academy/DatabaseInitializer.java

```diff
@@ -9,26 +9,30 @@
 import mate.academy.exception.DataProcessingException;
 
 public class DatabaseInitializer {
+    private static final String FILE_PATH = "src/main/resources/init_db.sql";
+
     public void createTable() {
         try (Connection connection = ConnectionUtil.getConnection();
                 Statement statement = connection.createStatement()) {
-            StringBuilder sqlCommands = new StringBuilder();
-            try (BufferedReader reader = new BufferedReader(
-                    new FileReader("src/main/resources/init_db.sql"))) {
-                String line;
-                while ((line = reader.readLine()) != null) {
-                    sqlCommands.append(line).append(" ");
-                }
-            } catch (IOException e) {
-                throw new DataProcessingException("Failed to read init_id.sql file: ", e);
-            }
-
-            statement.executeUpdate(sqlCommands.toString());
+            String sqlCommands = parseCommandFile(FILE_PATH);
```

Comment:
you shouldn't parse the file while the connection is opened. I'd recommend calling two methods inside your createTable: the first one to parse file and the second one to execute queries